infinispan-embedded/src/test/resources/infinispan-dist.xml

```diff
@@ -79,6 +84,9 @@
                 await-initial-transfer="true" />
         </distributed-cache>
 
+        <!-- Clustered Counters are defined at runtime by InfinispanDialect -->
+        <counters xmlns="urn:infinispan:config:counters:9.1" num-owners="1" reliability="CONSISTENT" />
```

Comment:
Let's not suggest to users a "num-owners=1", that's not very responsible ;)
Make it two in the documentation at least?

+1 to keep it at one in the integration tests.

Reply:
Let me clarify. I mean we need to add this line in the examples and make sure the documentation mentions it's needed.

Reply:
I completely agree